AppiumForMac/Server/Handlers/AfMHandlers.m

```diff
@@ -543,6 +543,9 @@ - (AppiumMacHTTPJSONResponse *)post_elements:(NSString*)path data:(NSData*)postD
                     [elements addObject:[NSDictionary dictionaryWithObject:myKey forKey:@"ELEMENT"]];
                 }
                 
+                return [AppiumMacHTTPJSONResponse responseWithJson:elements status:kAfMStatusCodeSuccess session:session.sessionId];
+                
+            } else {
```

Comment:
this `else` is, actually, not needed

Reply:
I think it is is better to respond with status `kAfMStatusCodeNoSuchElement` when no element is found.

Reply:
I mean you could just simplify it to 

```
...
return [AppiumMacHTTPJSONResponse responseWithJson:elements status:kAfMStatusCodeSuccess session:session.sessionId];
}
...
return [AppiumMacHTTPJSONResponse responseWithJson:nil status:kAfMStatusCodeNoSuchElement session:session.sessionId];
```

Reply:
Yes, you are right